scripts/ts/common/jiraTicket/index.ts

```diff
@@ -79,19 +79,16 @@ export const getJiraIdFromPrTitle = (
  * Try to retrieve Jira tickets from pr title
  * and transforms them into {@link GenericTicket}
  * @param title
+ * @returns a promise of {@link JiraTicketRetrievalResults} containing the jira tickets or an error. If no tickets are found, an empty array is returned
  */
 export const getTicketsFromTitle = async (
```

Comment:
```suggestion
export const getTicketsFromTitle = (
```
Do we still need `async` here?

Reply:
Nice catch, thank you. Given that we are returning a `Promise` from `getJiraTickets` and also a `Promise.resolve([])` we don't need to declare the function as `async`. Fixed in 7bcee7385be09f553d9c5c9ad78ff5deca23f793, thanks.